spec/rails_helper.rb

```diff
@@ -19,6 +19,7 @@
 
 # Stub Geocoder HTTP requests in specs
 Geocoder::DEFAULT_STUB_COORDINATES = [51.67014192630465, -1.2809649516211556].freeze
+Geocoder::DEFAULT_LOCATION = "TE5 T1NG".freeze
```

Comment:
Might this cause issues down the line not being a real postcode?

Reply:
What kind of issues?